client/packages/common/src/utils/formatters/formatters.ts

```diff
@@ -30,11 +30,16 @@ export const Formatter = {
   ): string => Papa.unparse(data, config),
   csvDateTimeString: (dateString?: string | null | undefined): string => {
     const date = dateString ? new Date(dateString) : null;
-    return date && isValid(date) ? format(date, "yyyy-MM-dd' 'HH:mm:ss") : '';
+    return date && isValid(date) ? format(date, "dd/MM/yyyy' 'HH:mm:ss") : '';
```

Comment:
Requested change from Richard and also consistent with other parts of UI..

Reply:
hm.